src/Malenki/Math/RandomComplex.php

```diff
@@ -157,7 +157,7 @@ protected static function checkOrder($float_min, $float_max)
     public function __get($name)
     {
         if (in_array($name, array('rho', 'theta', 'r', 'i'))) {
-            return $this->$name;
```

Comment:
Why this change? 

This break the possibility to have `$object->rho`, or `$object->theta` or `$object->r` or `$object->i` direct read access.

Reply:
I think directly return ```$name``` is appropriate.
If set the original code ```$this->$name```, the variable will return null.
Because this variable is not defined in this class.

Reply:
How about declare a ```private $name``` and change ```$this->$name``` to ```$this->name```?

Reply:
I see what you mean, but it is confusing, really.

But this class should be change in futur I think.

Reply:
I want keep magic getter to get this values, less confusing I think.